overview/introduction.md

```diff
@@ -12,7 +12,8 @@ You can expand the functionality of your Procore account by leveraging existing
 >
 > Procore API resources under the `/vapid` namespace were deprecated on February 1, 2021, and replaced by the new Rest v1.0 resources under the `/rest` namespace with a new architecture that supports versioning and expanded functionality.
 >
-> Although Procore will continue to support Vapid resources and address defects as needed, all new feature development for the Procore API will be done in Rest.
+> On 02/01/2022 the `/vapid` namespace will be sunset in accordance with our [API lifecycle guidelines](https://developers.procore.com/documentation/rest-api-lifecycle).
```

Comment:
I would suggest using a format like "February 1st, 2022" rather than "02/01/2022" because outside of the United States, this date would mean "January 2nd, 2022".

Reply:
Yes! That's a good point